content/developer/reference/backend/orm.rst

```diff
@@ -1249,6 +1251,12 @@ will yield::
 
     {'name': "A", 'description': "Extended"}
 
+.. warning:: if you use :attr:`~odoo.models.Model._inherit` with a list
+    but leaving out :attr:`~odoo.models.Model._name`, the class
+    name will be used to infer the model you're extending.
+    This is not the case when :attr:`~odoo.models.Model._inherit`
+    is a ``string``
```

Comment:
From the code I feel we should claim
`_inherit = 'str'` will set the default `Model._name` and is the only case we don't need to explicitly specify `Model._name`

So we can infer
```py
Class MyModel:
    # _name = 'my.model'
```
will automatically use `_name = 'my.model'` and log a warning, no matter if it has `_inherits = [...]` or not